mini_pupper_bringup/include/gait_config.h

```diff
@@ -1,16 +0,0 @@
-#ifndef GAIT_CONFIG_H
```

Comment:
So would we now rely on CHAMP defaults only? Or how would a user pass these settings (while using on a real robot)?

Reply:
I'm thinking of deleting this config file because it's not being used. Do you use this file? 
If someone are using these files, I'll move the files to mini_pupper_description.

Reply:
Ah, ok. If they are already in the description, we can remove these.